### PR TITLE
native integration landing

### PR DESCRIPTION
**Pull request by danieltprice**
Azure landing page:
https://neon-next-git-dprice-azure-landing-neondatabase.vercel.app/docs/manage/azure

Note: We might want to drop the links to the Azure CLI and SDK if they are not ready.

### CODE REVIEW

File: content/docs/manage/azure.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- **Consolidated billing**: Simplify cost management with unified billing through the Azure Marketplace.
+- **Integrated workflows**: Use the Azure CLI and SDK to manage Neon organizations as part of your regular workflows, integrated with your existing Azure resources.
+
+    <Admonition type="note">
+    Management of Neon projects, databases, and branches is supported through the Neon Console, CLI, and API. However, this public preview lays the groundwork for further deeper between Neon and Azure, including integration with other Azure native services.
+    </Admonition>
+
+### Get started
+
+To begin using **Neon as an Azure Native ISV Service**, refer to our [Azure Marketplace](/docs/introduction/billing-azure-marketplace) instructions. After you setup the integration, familiarize yourself with Neon by stepping through Neon's [Getting started](https://neon.tech/docs/introduction#get-started) guides. You can also manage your Neon Postgres Resource using the Azure CLI or SDK for .NET.
```
Comment (mocasii) [inline content/docs/manage/azure.md:31]: This link is failing for me

File: content/docs/manage/azure.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+<DetailIconCards>
+
+<a href="/docs/introduction/billing-azure-marketplace" description="Deploy Neon Postgres as Native ISV Service from the Azure Marketplace" icon="enable">Deploy Neon from Azure Marketplace</a>
```
Comment (mocasii) [inline content/docs/manage/azure.md:35]: This link too

File: content/docs/manage/azure.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/manage/azure.md:49]: separate PR

File: content/docs/manage/azure.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/manage/azure.md:53]: Separate PR

**Pull request merged**